Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.11-slim-bullseye
```

Comment:
Lets use something more common e.g. `python:3.11-alpine3.17`.

Reply:
Usage of python:3.11-alpine3.17 can decrease the size of the docker image, but I don't like alpine and try to avoid alpine due to symlinks to busybox and using musl instead of default glibc. Most of the pre-compiled binary wheels are compiled against glibc, and therefore Alpine disabled Linux wheel support.

And it's the reason why default python images (i.e. `python:3.11.1`, `python:3.11`, `python:3` and `python:latest`) are based on Debian Bullseye, please [take a look on Docker Hub](https://hub.docker.com/_/python). And the slim version of the python image is based on `debian:bullseye-slim`, so `python:3.11-slim-bullseye` is the same as `python:3.11.1-slim`, `python:3.11-slim` and `python:3-slim`

So, I think that `python:3.11-slim-bullseye` looks like the right choice. Anyway, I can change it to `python:3.11-slim` to avoid misreading.